packages/component/src/index.js

```diff
@@ -10,7 +11,7 @@ const { lazy } = loadableExports
 lazy.lib = libraryExports.lazy
 
 export default loadable
-export { lazy }
+export { lazy, loadableEvents }
```

Comment:
You are exposing `emit` to the end user, and this is something they should not use.
I like `loadableEvents.registerCallback` signature, but exposing internals is not great.